### PR TITLE
Publishing docs

### PR DESCRIPTION
**Pull request by logason**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝Changed routes:


| [**/workspace/publishing**](https://docs-git-publishing-docs.teamavo.now.sh/workspace/publishing) |
|:-:|
|<a href="https://docs-git-publishing-docs.teamavo.now.sh/workspace/publishing">Screenshot of /workspace/publishing</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/5PNZZCGDGOFqEwVSFkS8TWsYVuI_.png">(view full size)</a> |





1 deleted route:
- **/workspace/integrations**

Commit 709b19a551f935bc80477e4ec15dd63ca5d01618 (https://docs-git-publishing-docs.teamavo.now.sh).

File: pages/workspace/publishing.mdx [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+<center>
+  <img
+    src={require('../../images/auto_publishing.png')}
+    width="100%"
+    alt="Publish integration filter to publish only events sent from Web to Amplitude"
+  />
+</center>
+
+# Publishing integrations
```
Comment (hugihlynsson) [inline pages/workspace/publishing.mdx:53]: I find this title pretty confusing and the re-iteration of the list below it unhelpful.

File: pages/workspace/publishing.mdx [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+- <a href="#segment-protocols">Segment Protocols</a>
+- <a href="#mixpanel-lexicon">Mixpanel Lexicon</a>
+- <a href="#amplitude-govern">Amplitude Govern</a>
+- <a href="#webhook">Webhook</a>
+
+They enable your Tracking Plan to be
+
+- published with a push of a button
+- published whenever a branch is merged with Auto Publishing
+- downloaded from your Avo workspace on relevant format
```
Comment (hugihlynsson) [inline pages/workspace/publishing.mdx:66]: I love this section and feel like it's super valuable to understand this feature. Should we move it to the top?

File: pages/workspace/publishing.mdx [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+
+<img
+  src={require('../../images/publishing_protocols.svg')}
+  alt="Segment"
+  style={{ margin: '32px 0 -32px 0' }}
+/>
```
Comment (hugihlynsson) [inline pages/workspace/publishing.mdx:72]: Love the icons 😍

File: pages/workspace/publishing.mdx [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+
+## <a name="segment-protocols"></a> Segment Protocols
+
+The publish integration to [Segment Protocols](https://segment.com/docs/protocols) allows for your Tracking Plan to be published into Segment Protocols whenever branches are merged in your workspace or with a push of a button.
```
Comment (hugihlynsson) [inline pages/workspace/publishing.mdx:76]: Talking about "publish integrations" is confusing me here too.

File: pages/workspace/publishing.mdx [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+
+### Configure the Protocols integration
+
+To enable the direct publish integration with Segment Protocols, you need to configure it with the following parameters from Segment. This step is not neccassary if you only wish to download the Protocols JSON file from the Avo workspace.
```
Comment (hugihlynsson) [inline pages/workspace/publishing.mdx:80]: I'd love to know more about this step being unnecessary. What would I dow with the downloaded Protocols JSON file? Should we be explicit about there being an automatic way to use this integration and a manual way? Perhaps have an automatic section first and then end with a manual section?

File: pages/workspace/publishing.mdx [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+Available publish integrations are currently:
+
+- <a href="#segment-protocols">Segment Protocols</a>
+- <a href="#mixpanel-lexicon">Mixpanel Lexicon</a>
+- <a href="#amplitude-govern">Amplitude Govern</a>
+- <a href="#webhook">Webhook</a>
+
+## Publishing configuration
+
+### Filtering events for publishing
```
Comment (hugihlynsson) [inline pages/workspace/publishing.mdx:29]: Should we move this feature below the publishing types guides?

**Review by hugihlynsson — APPROVED** [review]
I read quickly thought this and it looks great for a first version. Do we want to mention auto-publishing and what that means somewhere? Perhaps next to the filtering feature?

File: pages/workspace/publishing.mdx (hunk 5 above)
Comment (logason) [reply pages/workspace/publishing.mdx:80]: Or maybe in this case just skip the mentioning of the manual step?

File: pages/workspace/publishing.mdx (hunk 6 above)
Comment (logason) [reply pages/workspace/publishing.mdx:29]: I'm not sure, I was wondering if they would be helpful in describing what "Publishing" means, before going into details of specific integrations. If we move them to the bottom I'm afraid no one will notice them, what do you think?

**Comment by logason** [comment]
> Do we want to mention auto-publishing and what that means somewhere? Perhaps next to the filtering feature?

Section about auto publishing should be in there already, next to the events filter, here: https://docs-git-publishing-docs.teamavo.now.sh/workspace/publishing#auto-publishing

File: pages/workspace/publishing.mdx (hunk 1 above)
Comment (logason) [reply pages/workspace/publishing.mdx]: Yes I agree, renamed to just "Integrations" in 759cbe4

File: pages/workspace/publishing.mdx (hunk 2 above)
Comment (logason) [reply pages/workspace/publishing.mdx]: Yes I think that would make sense, moved to top in 759cbe4 ✅

File: pages/workspace/publishing.mdx (hunk 3 above)
Comment (logason) [reply pages/workspace/publishing.mdx:65]: yeaas so pretty 😍

File: pages/workspace/publishing.mdx (hunk 4 above)
Comment (logason) [reply pages/workspace/publishing.mdx]: Hah yes that is definitely tricky, replaced all mentions of "publish integration" with just "integration" in 759cbe4, keep it simple

**Review by hugihlynsson — APPROVED** [review]
> Section about auto publishing should be in there already, next to the events filter, here: https://docs-git-publishing-docs.teamavo.now.sh/workspace/publishing#auto-publishing

I don't know how I missed that 🙈 Looks great! 🚀

**Pull request merged**